iphone/Classes/TiUITextWidget.m

```diff
@@ -18,6 +18,15 @@
 
 @implementation TiUITextWidget
 
+#ifdef TI_USE_AUTOLAYOUT
+-(void)initializeTiLayoutView
+{
+    [super initializeTiLayoutView];
+    [self setDefaultHeight:TiDimensionAutoSize];
+    [self setDefaultWidth:TiDimensionAutoFill];
```

Comment:
Yes, mentioned that we should do this for ProgressBar and TextField here:
https://jira.appcelerator.org/browse/TIMOB-20406